manifests/instance.pp

```diff
@@ -162,6 +162,7 @@
   Boolean                            $ssl                           = false,
   Elasticsearch::Status              $status                        = $elasticsearch::status,
   Optional[String]                   $system_key                    = $elasticsearch::system_key,
+  Integer                            $repo_version                  = $elastic_stack::repo::version,
```

Comment:
This line will generate a warning if users aren't leveraging the `elastic_stack` puppet module, or explicitly passing in a value. 

```
Warning: Unknown variable: 'elastic_stack::repo::version'. 
```

Not sure if this is acceptable, or whether to consider it as taking an implicit dependency on the elastic_stack module

Reply:
You’re right, I’ve not thought as much on how to handle correctly this version condition. I’ll update this when I have the time ti dig into. Or if someone as suggestions, I’ll consider it. Marked this as WIP